masonry/src/action.rs

```diff
@@ -18,7 +18,7 @@ pub enum Action {
     TextEntered(String),
     CheckboxChecked(bool),
     // FIXME - This is a huge hack
-    Other(Arc<dyn Any>),
+    Other(Arc<dyn Any + Send + Sync>),
```

Comment:
I made this change because it was already using `Arc` - if it was meant to be single threaded, it should have been `Rc`